latex/epoch.tex

```diff
@@ -1,54 +1,614 @@
-The pool reaping in rules are discribed in
-Figure~\ref{fig:ts-types:pool-reap} and Figure~\ref{fig:rules:pool-reap}.
+\newcommand{\UTxOEpEnv}{\type{UTxOEpEnv}}
```

Comment:
If we want the rules and associated definitions not crossing sections boundaries (which I think is is nicer), we could replace `\input` by `\include`, use the `htb` position specifier, and use `\clearpage` where appropriate.

Reply:
great, thanks!